### PR TITLE
fix(ui): Fix accessibility problems in Clusters

### PR DESCRIPTION
**Pull request by pedrottimark**
### Problems

1. Visit /main/clusters and then click a row

    Minor: Table header text should not be empty `<th class="" aria-label="Action"></th>`

2. Click **Next**

    Serious: Elements must meet minimum color contrast ratio

    * Element has insufficient color contrast of 4.32 (foreground color: #2c8820, background color: #edffeb, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1

        `<button type="button" class="mr-2 btn btn-success" data-testid="">`

    * Element has insufficient color contrast of 2.08 (foreground color: #5fb2f7, background color: #ebf6ff, font size: 9.0pt (12px), font weight: normal). Expected contrast ratio of 4.5:1

        `<button type="button" class="download uppercase text-primary-600 p-2 text-center text-sm border border-solid bg-primary-200 border-primary-300 hover:bg-primary-100 flex items-center">`

    * Element has insufficient color contrast of 4.32 (foreground color: #2c8820, background color: #edffeb, font size: 9.0pt (12px), font weight: normal). Expected contrast ratio of 4.5:1

        `<button class="border-2 font-600 inline-flex items-center justify-center rounded-sm uppercase text-sm border-success-500 bg-success-200 hover:bg-success-300 hover:text-success-800 text-success-700 p-2" type="button">Download Helm values</button>`

### Analysis and Solutions

1. Replace `Th` with `Td` element.

2. Replace the following with PatternFly `Button` element:

    * Upgrade and Delete buttons: `PanelButton` from src/Components

    * Next or Finish button: `PanelButton` from src/Components

    * Download YAML file and keys: HTML `button` element

    * Download Helm values: `SuccessButton` from ui-components package

3. Bonus to remove another leftover blue background from StackRox branding.

    * Remove `titleClassName` prop so collapsible cards have default header style.

## Checklist
- [ ] Investigated and inspected CI test results
- [x] ~Unit test and regression tests added~

## Testing Performed

1. `yarn lint` in ui
2. `yarn build` in ui and then compute branch - master for the following
    * `wc build/static/js/*.js`
        main.js -173 = 4231696 - 4231869
        total -1848 = 10310740 - 10312588
    * `ls -al build/static/js/*.js | wc`
        files 0 = 80 - 80
3. `yarn start` in ui

### Manual testing

1. Visit /main/clusters
    * See **Upgrade** and **Delete** buttons disabled in first picture, and with **Delete** enabled in second picture.
        clusters
        clusters_Delete

2. Click a row to open side panel.
    * See **Next** button.
        clusters_Next

3. Click **Next**.
    * See **Finish** button.
    * See collapsible headings with default heading style.
    * See both **Download** buttons, and then click to see disabled and loading state (temporary change to leave it spinning long enough to take the second picture).
        clusters_Finish
        clusters_Download

### Integration testing

1. `yarn cypress-open` in ui/apps/platform
    * clusters/clusterDeletion.test.js
    * clusters/clusters.test.js
    * clusters/clusterCertificateExpiration.test.js
    * clusters/clusterHealthStatus.test.js
    * clusters/clustersManager.test.js
    * clusters/delegateScanning.test.js
    * clusters/redirectFromDashboard.test.js

### CODE REVIEW

File: ui/apps/platform/src/Containers/Clusters/ClusterDeployment.js [hunk 1]
```diff
@@ -62,20 +60,15 @@ const ClusterDeploymentPage = ({
                                     />
                                 </div>
                                 <div className="flex justify-center px-3">
-                                    {isDownloadingBundle ? (
-                                        <Spinner isSVG size="lg" />
-                                    ) : (
```
Comment (vjwilson) [inline ui/apps/platform/src/Containers/Clusters/ClusterDeployment.js:67]: Is the conditional rendering no longer needed?

File: ui/apps/platform/src/Containers/Clusters/ClusterLabelsTable.tsx [hunk 2]
```diff
@@ -77,7 +77,7 @@ function ClusterLabelsTable({
                 <Tr>
                     <Th>Key</Th>
                     <Th>Value</Th>
-                    {hasAction && <Th aria-label="Action" />}
+                    {hasAction && <Td />}
```
Comment (vjwilson) [inline ui/apps/platform/src/Containers/Clusters/ClusterLabelsTable.tsx:80]: This seems unsemantic. Wouldn't having some kind of label be more appropriate a fix?

**Review by vjwilson — APPROVED** [review]
Just 2 concerns to check

File: ui/apps/platform/src/Containers/Clusters/ClusterDeployment.js (hunk 1 above)
Comment (pedrottimark) [reply ui/apps/platform/src/Containers/Clusters/ClusterDeployment.js:67]: Good question. Superseded by props of `Button` element that are illustrated in the last picture.

```tsx
isDisabled={isDownloadingBundle}
isLoading={isDownloadingBundle}
```

File: ui/apps/platform/src/Containers/Clusters/ClusterLabelsTable.tsx (hunk 2 above)
Comment (pedrottimark) [reply ui/apps/platform/src/Containers/Clusters/ClusterLabelsTable.tsx:80]: Yes, good point. I will change it render visible **Action** table heading.

**Comment by pedrottimark** [comment]
ocp-4-13-ui-e2e-tests failure

> Cluster initialization failed because one or more operators are not functioning properly.

**Pull request merged**